### ISSUE TITLE
Issue of contract parameter type

### ISSUE THREAD

**Issue by ProDog** [neo-project/neo-devpack-dotnet#357]
**Describe the bug**
Execution `FAULT` when the contract uses the byte[] parameter.

**To Reproduce**
Contract:
```C#
        public static object Test2(byte[] a)
        {
            return a;           
        }
```
Invoke:
```
invoke 0x72d660435e3ca79c9ebdd8274db4c2665b7c438a Test2 [{"type":"ByteArray","value":"AQI="}]
```
Result:
```
Invoking script with: '0c02010211c00c09626f6f6c54657374320c148a437c5b66c2b44d27d8bd9e9ca73c5e4360d67241627d5b52'
VM State: FAULT
Gas Consumed: 0.0127913
Result Stack: []
Error: Invalid type for SETITEM: ByteString
Error: Invalid type for SETITEM: ByteString
```

**Platform:**
 - OS: [Windows 10 x64]
 - Version [neo master]

**Comment by shargon** [comment]
@ProDog could you check if it was solved, in https://github.com/neo-project/neo-devpack-dotnet/pull/360 it seems that it works

**Comment by ProDog** [comment]
> @ProDog could you check if it was solved? in #360 it seems that it works

Yes, It's been fixed.
